eagleproject/core/migrations/0048_day_alter_analyticsreport_created_at_and_more.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 4.1.1 on 2023-02-12 12:33
```

Comment:
@rolandpo Can you delete this file from this PR? I have noticed that the changes this migration makes is already on prod DB. So committing them makes the prod deployment to fail

Reply:
yep, deleted

Reply:
Thank you